src/player.rs

```diff
@@ -430,7 +450,59 @@ impl PlayerInternal {
         let key = self.session.audio_key().request(track.id, file_id).wait().unwrap();
 
         let encrypted_file = AudioFile::open(&self.session, file_id).wait().unwrap();
-        let audio_file = Subfile::new(AudioDecrypt::new(key, encrypted_file), 0xa7);
+
+        let mut decrypted_file = AudioDecrypt::new(key, encrypted_file);
+
+        let mut normalization_factor: f32 = 1.0;
+
+        if self.config.normalization {
+            //buffer for float bytes
+            let mut track_gain_float_bytes = [0; 4];
+
+            decrypted_file.seek(SeekFrom::Start(144)).unwrap(); // 4 bytes as LE float
+            decrypted_file.read(&mut track_gain_float_bytes).unwrap();
```

Comment:
You should use [`read_f32::<LittleEndian>`](https://docs.rs/byteorder/1.2.1/byteorder/trait.ReadBytesExt.html#method.read_f32)